blob/s3blob/s3blob_test.go

```diff
@@ -476,6 +476,18 @@ func TestOpenBucketFromURL(t *testing.T) {
 		{"s3://mybucket?awssdk=v2", false},
 		// OK, use KMS Server Side Encryption
 		{"s3://mybucket?ssetype=aws:kms&kmskeyid=arn:aws:us-east-1:12345:key/1-a-2-b", false},
+		// OK, use S3 Transfer acceleration
+		{"s3://mybucket?accelerate=true", false},
+		// OK, use dual stack endpoints
+		{"s3://mybucket?dualstack=true", false},
```

Comment:
Nit: You don't necessarily need to test all the combinations of things that are supported in aws.go. Just one is OK, plus "accelerate" since it's specific to s3blob.

Reply:
Ok. AWS v2 is unhappy if `accelerate=true&fips=true`, I believe:

```
operation error S3: ListObjectsV2, resolve auth scheme: resolve endpoint: endpoint rule error, Accelerate cannot be used with FIPS
```